tasks/clientlibs.js

```diff
@@ -220,16 +224,17 @@ module.exports = function (grunt) {
 							clientLibCSS = getIncludeString(clientLibName, 'css');
 
 							// Sort the array into dependency order
-							clientLibCSSObj = performSort(clientLibCSSObj);
+							clientLibCSSOrderObj = performSort(clientLibCSSObj);						
 
 							// Generate and minify the string
-							for (var j = 0; j < clientLibCSSObj.length; j = j + 1) {
-								clientLibCSS += grunt.file.read(clientLibCSSObj[j].fileName, { encoding: 'utf8' }) + '\r\n';
-
+							clientLibIncludeOrderObjCSS = [];
+							for (var j = 0; j < clientLibCSSOrderObj.length; j = j + 1) {
```

Comment:
Probably create the variable for "clientLibCSSObj"/"clientLibCSSOrderObj" length so you're not having to call the length property each time?